client/src/app/components/MessageButton.scss

```diff
@@ -46,6 +46,16 @@
     }
   }
 
+  .message-label--bottom {
```

Comment:
What's the difference between `.message-button--message` and `.message-label` ? 
Looks to me like they are the same thing. So why call them differently ?
This should probably be just `bottom` and `top` and nested inside `.message-button--message` as it's always attached after that class, never alone.

Reply:
`.message-button--message` has the complete styles for the copy button like its color, animation and transform etc and `.message-label--bottom` and `.message-label--top` are only used to override the `.message-button--message` styles for `top` and `right` properties.
I can change there name but it cannot be nested imho.

Reply:
Yes, you can nest them with the `&` scss operator. I think nesting them is better cause it makes it clearer that those classes are tied together.